packages/components-native/src/ThumbnailList/ThumbnailList.tsx

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import { View } from "react-native";
+import isNil from "lodash/isNil";
+import { styles } from "./ThumbnailList.style";
+import { FormatFile } from "../FormatFile";
+import { CreateThumbnail, File } from "../FormatFile/types";
+
+export enum RowCount {
+  TwoRows = 2,
+  ThreeRows = 3,
+}
+
+export interface ThumbnailListProps {
```

Comment:
Should we put the above `enum` and this `interface` in a separate `types.ts` file?  It's not like this file is too big but just wondering if you think it's worth it based on patterns of other components.

Reply:
Makes sense to me. I can update it